deltachat-ios/View/ContactCell.swift

```diff
@@ -191,6 +230,16 @@ class ContactCell: UITableViewCell {
     }
 
     func setStatusIndicators(unreadCount: Int, status: Int, visibility: Int32, isLocationStreaming: Bool, isMuted: Bool) {
+        if isLargeText {
```

Comment:
for very large fonts (available from accessibilty menu) only the unread message counter will be shown. all other icons are to small anyways for visually impaired ppl and reduce the amount of readable characters of the chat title and the subtitle (last message)